### PR TITLE
Adjust config

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: config/console.php [hunk 1]
```diff
@@ -17,10 +18,7 @@
         return new Debugger(
             $container->get(DebuggerIdGenerator::class),
             $container->get(StorageInterface::class),
-            array_map(
-                static fn ($class) => $container->get($class),
-                array_merge($params['collectors'], $params['collectors.console'] ?? [])
-            )
+            DynamicReferencesArray::from(array_merge($params['collectors'], $params['collectors.console'] ?? []))
```
Comment (yiiliveext) [inline config/console.php:21]: Dynamic references don't work inside callables.

Comment (rustamwin) [reply config/console.php]: Done.

File: config/console.php [hunk 2]
```diff
@@ -2,25 +2,23 @@
...
-                array_merge($params['collectors'], $params['collectors.console'] ?? [])
-            )
-        );
-    },
+    Debugger::class => [
+        '__construct()' => [
+            'collectors' => ReferencesArray::from(
+                array_merge(
+                    $params['yiisoft/yii-debug']['collectors'],
+                    $params['yiisoft/yii-debug']['collectors.web'] ?? []
```
Comment (xepozz) [inline config/console.php:18]: `collectors.web` – is it a typo?

Comment (rustamwin) [reply config/console.php:18]: No. https://github.com/yiisoft/yii-debug/blob/9dccc68c34c43a12ec0eac13fbc492a361f78465/config/params.php#L37-L42

Comment (samdark) [reply config/console.php:18]: Should it be `collectors.console`?

Comment (rustamwin) [reply config/console.php]: Fixed.

**Review by samdark — APPROVED** [review]

**Pull request merged**